blazor-application/wwwroot/index.html

```diff
@@ -8,13 +8,17 @@
     <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
     <link href="css/app.css" rel="stylesheet" />
     <link href="BlazorBLE.styles.css" rel="stylesheet" />
+    <title>Tailwind via Play CDN</title>
```

Comment:
Intentional?

Reply:
Nope, leftover from the guide I used - oops